brainstormforce/astra
# Alignment & Toggle Issue with Editor Icon

### Description

Fixed two cases:
1. Editor icon toggle show when already disabled.
2. CSS icon alignment fix when icon docked in alignment with title.

### Screenshots

https://d.pr/v/cmmrXg

### Types of changes




bugfix
### How has this been tested?

- Test icon toggle on/off.
- Publish and check in frontend.
- Check the alignment when editor stretched.
- Check the alignment when editor contrained.

### Checklist:
